src/metabase/feature_extraction/insights.clj

```diff
@@ -0,0 +1,225 @@
+(ns metabase.feature-extraction.insights
```

Comment:
@sbelak I couldn't find the tests covering this ns, could you point me to them?

Reply:
+1

Reply:
added here https://github.com/metabase/metabase/pull/6266/files#diff-b4d18e3f6af9ee0f6fc3ca4160d8f254

Reply:
Oh ok I see tests have been added at metabase.feature-extraction.insights-tests